public/app/features/explore/hooks/useExplorePageTitle.ts

```diff
@@ -19,8 +18,19 @@ export function useExplorePageTitle(params: ExploreQueryParams) {
       return;
     }
 
+    let panesObject: unknown;
```

Comment:
Could we modify `safeParseJson` instead? Or at least delete it? I think that was the only place it was used.

Reply:
i didn't notice it wasn't used anymore, honestly i'd rather remove it as there's nothing "safe" about it imho